jcli/src/jcli_app/vote/tally/mod.rs

```diff
@@ -12,18 +12,34 @@ pub enum Tally {
     /// The decryption share data will be printed in hexadecimal encoding
     /// on standard output.
     DecryptionShare(decryption_tally::TallyGenerateDecryptionShare),
+    /// Create a decryption share for private voting tally.
+    ///
+    /// The decryption share data will be printed in hexadecimal encoding
+    /// on standard output.
+    VotePlanDecryptionShares(decryption_tally::TallyGenerateVotePlanDecryptionShares),
+    /// Merge multiple sets of shares in a single object to be used in the
+    /// decryption of a vote plan.
+    MergeShares(decryption_tally::MergeShares),
     /// Decrypt a tally with decryption shares.
     ///
     /// The decrypted tally data will be printed in hexadecimal encoding
     /// on standard output.
     Decrypt(decrypt_shares::TallyDecryptWithAllShares),
+    /// Decrypt all proposals in a vote plan.
+    ///
+    /// The decrypted tally data will be printed in hexadecimal encoding
+    /// on standard output.
+    DecryptVotePlan(decrypt_shares::TallyVotePlanWithAllShares),
```

Comment:
The name is a bit misleading: we're decrypting tally results, not the vote plan.
`tally decrypt-results` and deprecate `tally decrypt`?